src/core/core.scale.js

```diff
@@ -355,12 +356,19 @@ var Scale = Element.extend({
 
 	_getMinMax: function(canStack) {
 		var me = this;
-		var metas = me._getMatchingVisibleMetas();
 		var min = Number.POSITIVE_INFINITY;
 		var max = Number.NEGATIVE_INFINITY;
 		var minPositive = Number.POSITIVE_INFINITY;
-		var i, ilen, minmax;
+		var i, ilen, metas, minmax;
 
+		if (isFinite(me._userMin) && isFinite(me._userMax)) {
```

Comment:
What if only one is? Then should we return that value below?

Reply:
done